tests/RobotFramework/tests/tedge_agent/workflows/custom_operation.robot

```diff
@@ -26,6 +26,10 @@ Override Built-In Operation
     Should Contain      ${software_list[0]}    "currentSoftwareList"
     Should Contain      ${software_list[0]}    "mosquitto"
     Should Contain      ${software_list[0]}    "tedge"
+    Should Contain      ${software_list[0]}    "preprocess"
+    Should Contain      ${software_list[0]}    "ready"
```

Comment:
This check fails despite the preprocess step being actually executed. Indeed, the final payload `'{"currentSoftwareList":...,"postprocess":"done","status":"successful"}'` does not contain the `"preprocess"` key. However, as witnessed by the `"postprocess":"done"`  and `"status":"successful"` values, the command went successfully over all the steps in sequence (init -> scheduled -> preprocess -> executing -> postprocess -> successful).

What is going on? The issue is that the builtin actions remove any unknown field. This has to be fixed.